### PR TITLE
Adding Mapbox Isochrone API example

### PR DESCRIPTION
**Pull request by langsmith**
Adds an example of using [the Mapbox Isochone API](https://www.mapbox.com/api-documentation/#isochrone). There is no Isochrone wrapper in [the Mapbox Java SDK](https://github.com/mapbox/mapbox-java) at this time, but the API can still be interacted with.

ezgif com-resize 1

**Examples of other styling options. _Not built into the example though_**
custom color fill iso
rainbow iso

### CODE REVIEW

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/IsochoneActivity.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    });
+  }
+
+  private void initIsochroneGeoJsonSource() {
+    try {
+      // Retrieve GeoJSON information from the Mapbox Isochrone API
+      GeoJsonSource source = new GeoJsonSource(GEOJSON_SOURCE_ID, new URL("https://api.mapbox.com/isochrone/v1/"
+        + "mapbox/driving/" + "-122.43817518306757,37.762834146042294?access_token="
+        + getString(R.string.access_token) + "&polygons=false&" + "contours_minutes=5,10,15&contours_colors"
+        + "=6706ce,04e813,4286f4"));
```
[the remark this answers is not in the document]
Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/IsochoneActivity.java]: Done!

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/IsochoneActivity.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  }
+
+  private void initFillLayer() {
+    // Create and style a FillLayer based on information in the Isochrone API response
+    FillLayer isochoneFillLayer = new FillLayer("polygon", GEOJSON_SOURCE_ID);
+    isochoneFillLayer.setProperties(
+      fillColor(get("color")),
+      fillOpacity(get("fillOpacity")));
+    isochoneFillLayer.setFilter(eq(geometryType(), literal("Polygon")));
+    isochoneFillLayer.setFilter(eq(literal("$type"), literal("Polygon")));
```
[the remark this answers is not in the document]
Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/IsochoneActivity.java]: Fixed

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/IsochoneActivity.java [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        initFillLayer();
+        initLineLayer();
+      }
+    });
+  }
+
+  private void initIsochroneGeoJsonSource() {
+    try {
+      // Retrieve GeoJSON information from the Mapbox Isochrone API
+      GeoJsonSource source = new GeoJsonSource(GEOJSON_SOURCE_ID, new URL("https://api.mapbox.com/isochrone/v1/"
```
[the remark this answers is not in the document]
Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/IsochoneActivity.java]: Done! It's working great 😄

**Comment by zugaldia** [comment]
@osana could you give this example a final review?

**Pull request merged**